A/Arb/build_tarballs.jl

```diff
@@ -22,24 +22,23 @@ import Pkg.Types: VersionSpec
 # to all components.
 
 name = "Arb"
-upstream_version = v"2.21.1"
-build_for_julia16_or_newer = true
-version_offset = build_for_julia16_or_newer ? v"0.0.1" : v"0.0.0"
+upstream_version = v"2.22.0"
+version_offset = v"0.0.1"
 version = VersionNumber(upstream_version.major * 100 + version_offset.major,
                         upstream_version.minor * 100 + version_offset.minor,
                         upstream_version.patch * 100 + version_offset.patch)
 
+# Collection of sources required to complete build
 # Collection of sources required to complete build
 sources = [
     ArchiveSource("https://github.com/fredrik-johansson/arb/archive/refs/tags/$(upstream_version).tar.gz",
-                  "aecc9f55eb35a00a2490e9a7536a0babf8ac86bb13d32a4a95e1216f9f5cbfa8"),
+                  "c9903d49a3e89bbd0ae114eeed97eaea3877e0e1a31a5cdb24ba0c84e4e81f4a")
 ]
```

Comment:
maybe switching to git source is more convenient?
```julia
sources = [
    GitSource("https://github.com/fredrik-johansson/arb.git", "c2b7e36915a451455c1f115ce343dd85bdc30c59")
]
```
where we directly reference the tagging commit.

But this is just an option to consider.